docs/design/README.md

```diff
@@ -347,12 +383,27 @@ selected.
 
 ### String types
 
+> **Note:** This is provisional, no design for string types has been through the
+> proposal process yet.
```

Comment:
I am in the process of making that change. It was unclear to me if I should be adding the provisional marks to this section while I'm touching it, or do all the provisional marks in a specific change.

Reply:
My own suggestion would be to try to add them comprehensively, rather than piecemeal. Having them in a few spots implies the rest _is_ firmed up.

Reply:
Okay, will do. Just to note, I had a similar concern at a smaller scale here. Since I'm adding references, I was worried that strings themselves might look more finished than they are.

Reply:
I think this is a consequence of not having the broken-out files for everything.

Without those, we can't treat this file as comprehensively provisional, and just do the more nuanced documentation there. Is this having the fine grained provisionality going to be OK in order to allow us to mix provisional and decided things at this layer?

Reply:
I think that's kind of the state of things, that the design overview looks more finished than it is, not just for strings. I think I'm prejudiced though since I'd rather keep references in the individual designs, rather than copying some into the main readme.

Reply:
Looking at the deleted primitive_types.md, even the individual designs have the problem of mixing provisional and decided.

Reply:
In what way? It has a TODO at the top. I think when it was last updated, everything was provisional.

Reply:
It started out completely provisional, but subsequent proposals have updated it.

Reply:
Example: https://github.com/carbon-language/carbon-lang/pull/1901/files#diff-aea41795f576fecec3f8ba28d42efac82bcce528df637b06dc4ea7ddc1858a16